packages/account/src/Sections/Security/ConnectedApps/connected-apps.tsx

```diff
@@ -1,163 +1,101 @@
+                                    data_source={connected_apps as unknown as TSource[]}
+                                    columns={DataTableTemplate(handleToggleModal)}
+                                    content_loader='span'
+                                />
+                            </div>
+                        )}
+                    </div>
                 ) : (
-                    <React.Fragment>
-                        <DesktopWrapper>
-                            <DataTable
-                                className='connected-apps'
-                                data_source={connected_apps}
-                                columns={GetConnectedAppsColumnsTemplate(handleToggleModal)}
-                            />
-                        </DesktopWrapper>
-                        <MobileWrapper>
-                            <DataList
-                                className='connected-apps'
-                                data_source={connected_apps}
-                                row_gap={is_appstore ? 16 : 10}
-                                rowRenderer={mobileRowRenderer}
-                            />
-                        </MobileWrapper>
-                    </React.Fragment>
+                    <ConnectedAppsEmpty />
                 )}
-
-                {!is_loading && !!connected_apps.length && <ConnectedAppsArticle />}
-            </div>
-
-            <Modal is_open={is_modal_open} className='connected-apps' toggleModal={handleToggleModal}>
+            </section>
+            <section className='connected-apps__articles--wrapper'>
+                <ConnectedAppsKnowMore />
+                <ConnectedAppsEarnMore />
+            </section>
+            <Modal is_open={is_modal_open} className='connected-apps' width='44rem'>
                 <Modal.Body>
-                    <div className='connected-app-modal'>
-                        <Icon
-                            icon={is_appstore ? 'IcAccountTrashCanDashboard' : 'IcAccountTrashCan'}
-                            size={128}
-                            className='connected-app-modal__icon'
-                        />
-                        <Text as='p' color='prominent' weight='bold'>
-                            {localize('Confirm revoke access?')}
-                        </Text>
-                        <div
-                            className={classNames('connected-app-modal__confirmation', {
-                                'connected-app-modal__confirmation-dashboard': is_appstore,
-                            })}
-                        >
-                            <Button secondary onClick={handleToggleModal}>
-                                {localize('Back')}
+                    <div className='connected-apps-modal--wrapper'>
+                        <div className='connected-apps-modal--icon'>
+                            <Icon icon='IcAccountTrashCan' size={128} />
```

Comment:
clarification: how will be the mobile view here?

Reply:
like this. you asked about this, right?
![Screenshot 2023-10-12 at 10 42 35 AM](https://github.com/binary-com/deriv-app/assets/122449658/a2428735-de94-42be-9bb2-c787bdb19018)